descriptors/ModuleDescriptor-template.json

```
@@ -49,17 +49,15 @@
         {
           "methods": ["GET"],
           "pathPattern": "/eholdings/kb-credentials/{credentialsId}/users",
-          "permissionsRequired": ["kb-ebsco.kb-credentials.users.collection.get"]
+          "permissionsRequired": ["kb-ebsco.kb-credentials.users.collection.get"],
+          "modulePermissions": ["users.collection.get",
+          "usergroups.collection.get"]
```

It should be in one line:
`"modulePermissions": ["users.collection.get","usergroups.collection.get"]`
Or separate lines, like here:
```suggestion
          "modulePermissions": [
            "users.collection.get",
            "usergroups.collection.get"
          ]
```